engine/src/main/java/com/datatorrent/stram/webapp/TypeGraph.java

```diff
@@ -355,7 +356,8 @@ public int size()
     }
     Set<String> result = new TreeSet<>();
     for (TypeGraphVertex node : tgv.allInstantiableDescendants) {
-      if ((isAncestor(InputOperator.class.getName(), node.typeName) || !getAllInputPorts(node).isEmpty())) {
+      if ((isAncestor(InputOperator.class.getName(), node.typeName) || isAncestor(Module.class.getName(), node.typeName)
```

Comment:
You can rename the method to getAllDTInstantiableGenericOperators
And also in the OperatorDiscoverer, you should rename loadOperatorClass to loadGenericOperatorClasses